sdk/helper/automatedrotationutil/fields.go

```diff
@@ -104,3 +114,77 @@ func AddAutomatedRotationFields(m map[string]*framework.FieldSchema) {
 		m[name] = schema
 	}
 }
+
+// HandleRotationJobOperation is a helper method wrapper to the two individual helper methods
+// that register and deregister a rotation job each, and returns a standardized error response
+// with a warning in the event of an unexpected error.
+// revertStorageFunc is a function that should revert the original config back in storege before
+// any fields were updated.
+func (p *AutomatedRotationParams) HandleRotationJobOperation(ctx context.Context, b *framework.Backend, req *logical.Request, revertStorageFunc func() error) *logical.Response {
+	if p.DisableAutomatedRotation {
```

Comment:
Should we also move this `if` check into `HandleDeregisterRotationJob`?

Reply:
I think we actually need to do the opposite and put it on the caller to check this when using the individual helpers. So this wrapper helper method should check both conditionals. Gonna include this change for review